docs/resources/billingconductor_pricing_rule.md

```diff
@@ -10,7 +9,120 @@ description: |-
 
 A markup/discount that is defined for a specific set of services that can later be associated with a pricing plan.
 
+## Example Usage
+
+### First example
```

Comment:
same here, change the boilerplate section header
for example `### Pricing rule with global scope`